.gitlab-ci.yml

```diff
@@ -68,7 +68,7 @@ build:
 
 build-wasm-peregrine:
   image:
-    name: paritytech/srtool:1.60.0
+    name: paritytech/srtool:1.61.0
```

Comment:
Wasn't it 1.62.0?

Reply:
Good catch! The Polkadot release was built with `1.61.0` which is why I bumped to that. However, my notes mentioned `1.62.0` which is the latest version. I bumped to that 55bda19.